### PR TITLE
[docs] added mirantis integrations

### LINKED ISSUE #16564: [Docs] Yugabyte-DB deployment on Mirantis Kubernetes Engine
### Description

The steps to deploy a single node YugabyteDB cluster on MKE are mentioned in this [doc](https://docs.google.com/document/d/1BBi1vdJGIIu8MKuhUoquBl4vfdtX3tsX6wGJSvzl95A/edit). This needs to be added to YugabyteDB documentation.

### Warning: Please confirm that this issue does not contain any sensitive information

- [X] I confirm this issue does not contain any sensitive information.

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves #16564 
Build preview : https://deploy-preview-17044--infallible-bardeen-164bc9.netlify.app/preview/integrations/mirantis/

### CODE REVIEW

File: docs/content/preview/integrations/_index.md [hunk 1]
```diff
@@ -124,6 +124,12 @@ For information on YugabyteDB-compatible drivers and ORMs, refer to [Drivers and
     </a>
   </li>
 
+   <li>
+    <a href="mirantis/">
+      <img src="/images/section_icons/develop/ecosystem/mirantis.png">
+      Mirantis
```
Comment (HarshDaryani896) [inline docs/content/preview/integrations/_index.md:130]: nit : Change this to `MKE` or `Mirantis MKE`

**Review by HarshDaryani896 — APPROVED** [review]
LGTM. Thanks @aishwarya24

File: docs/content/preview/integrations/mirantis.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+- Robust and scalable applications deployment
+- Multi-tenant software offerings
+
+The following sections describe how to deploy a single node YugabyteDB cluster on Mirantis MKE using [kubectl](https://kubernetes.io/docs/reference/kubectl/) and [helm](https://helm.sh/).
+
+This page describes the steps for a single node cluster for the purpose of simplicity, as you require more than one machine/VM for a multi node cluster deployment.
+
+## Prerequisite
+
+Before installing a single node YugabyteDB cluster using Mirantis, ensure that you have the Docker runtime installed on your localhost. To download and install Docker, select one of the following environments:
```
Comment (HarshDaryani896) [inline docs/content/preview/integrations/mirantis.md:27]: Rephrase `on your localhost.` to something like `on the host on which you are installing MKE.`

File: docs/content/preview/integrations/mirantis.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+<i class="fa-brands fa-apple" aria-hidden="true"></i> [Docker for Mac](https://store.docker.com/editions/community/docker-ce-desktop-mac)
+
+<i class="fa-brands fa-centos"></i> [Docker for CentOS](https://store.docker.com/editions/community/docker-ce-server-centos)
+
+<i class="fa-brands fa-ubuntu"></i> [Docker for Ubuntu](https://store.docker.com/editions/community/docker-ce-server-ubuntu)
+
+<i class="icon-debian"></i> [Docker for Debian](https://store.docker.com/editions/community/docker-ce-server-debian)
+
+<i class="fa-brands fa-windows" aria-hidden="true"></i> [Docker for Windows](https://store.docker.com/editions/community/docker-ce-desktop-windows)
```
Comment (HarshDaryani896) [inline docs/content/preview/integrations/mirantis.md:37]: Remove `Docker for Windows`

File: docs/content/preview/integrations/mirantis.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+  preview_integrations:
+    identifier: mirantis
+    parent: integrations
+    weight: 571
+type: docs
+---
+
+[Mirantis Kubernetes Engine (MKE)](https://docs.mirantis.com/mke/3.5/overview.html) is a container orchestration platform for developing and running modern applications at scale, on private clouds, public clouds, and on bare metal.
+
+MKE as a container orchestration platform is specially beneficial in the following scenarios:
```
Comment (ddhodge) [inline docs/content/preview/integrations/mirantis.md:15]:
```suggestion
MKE as a container orchestration platform is especially beneficial in the following scenarios:
```

File: docs/content/preview/integrations/mirantis.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+
+- Orchestrating more than one container
+- Robust and scalable applications deployment
+- Multi-tenant software offerings
+
+The following sections describe how to deploy a single node YugabyteDB cluster on Mirantis MKE using [kubectl](https://kubernetes.io/docs/reference/kubectl/) and [helm](https://helm.sh/).
```
Comment (ddhodge) [inline docs/content/preview/integrations/mirantis.md:21]:
```suggestion
The following sections describe how to deploy a single-node YugabyteDB cluster on Mirantis MKE using [kubectl](https://kubernetes.io/docs/reference/kubectl/) and [helm](https://helm.sh/).
```

File: docs/content/preview/integrations/mirantis.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+
+This page describes the steps for a single node cluster for the purpose of simplicity, as you require more than one machine/VM for a multi node cluster deployment.
```
Comment (ddhodge) [inline docs/content/preview/integrations/mirantis.md:23]:
```suggestion
This page describes the steps for a single-node cluster for the purpose of simplicity, as you require more than one machine/VM for a multi-node cluster deployment.
```

File: docs/content/preview/integrations/mirantis.md [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+
+    ```sh
+    docker container run --rm -it --name ucp \
+          -v /var/run/docker.sock:/var/run/docker.sock \
+          mirantis/ucp:3.5.8 install \
+          --host-address <node-ip> \ // Replace <node-ip> with the IP address of your machine.
+          --interactive
+    ```
+
+    When prompted, enter the `username` and `password` that you want to set which will be used to access MKE web UI.
```
Comment (ddhodge) [inline docs/content/preview/integrations/mirantis.md:53]:
```suggestion
    When prompted, enter the `username` and `password` that you want to set; these are used to access the MKE web UI.
```

File: docs/content/preview/integrations/mirantis.md [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+            path: "/mnt/data"
+        ```
+
+    1. Apply the configuration using the following command:
+
+        ```sh
+        kubectl apply -f volume.yaml
+        ```
+
+    1. Repeat the preceding two steps to create the remaining PersistentVolumes (PV) : `task-pv-volume2`, `task-pv-volume3`, and `task-pv-volume4` by changing the metadata name in `volume.yaml` for each volume.
```
Comment (ddhodge) [inline docs/content/preview/integrations/mirantis.md:120]: acronym defined but not the first use and not used elsewhere

all these added PVs are added to the same volume.yaml file?

Comment (ddhodge) [inline docs/content/preview/integrations/mirantis.md:120]:
```suggestion
    1. Repeat the preceding two steps to create the remaining PVs: `task-pv-volume2`, `task-pv-volume3`, and `task-pv-volume4` by changing the metadata name in `volume.yaml` for each volume.
```

File: docs/content/preview/integrations/mirantis.md [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+    ```sh
+    kubectl get storageclass
+    ```
+
+    ```output
+    NAME                 PROVISIONER                   RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+    yb-storage(default)  kubernetes.io/no-provisioner  Delete          WaitForFirstConsumer   false                  23s
+    ```
+
+1. Create four [PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) using the following steps:
```
Comment (ddhodge) [inline docs/content/preview/integrations/mirantis.md:93]:
```suggestion
1. Create four [PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)(PVs) using the following steps:
```

File: docs/content/preview/integrations/mirantis.md (hunk 5 above)
Comment (ddhodge) [reply docs/content/preview/integrations/mirantis.md:21]: hyphens

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/integrations/mirantis.md (hunk 8 above)
Comment (aishwarya24) [reply docs/content/preview/integrations/mirantis.md:120]: I thought so . I actually had the same doubt. @HarshDaryani896 - can you clarify this please ?

Comment (ddhodge) [reply docs/content/preview/integrations/mirantis.md]: so, change the metadata name and re-run the command on the same file, x3

**Pull request merged**